scripts/generateLitTests.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const glob = require('glob');
+
+const packages = fs.readdirSync('packages');
+
+packages
+  .filter((pkg) => {
+    return fs.existsSync(`packages/${pkg}/vaadin-lit-${pkg}.js`);
+  })
+  .flatMap((pkg) => {
+    return glob.sync(`packages/${pkg}/test/*.test.{js,ts}`);
+  })
+  .filter((testPath) => {
+    return !/(-polymer|-lit)(\.generated)?\.test/u.test(testPath);
+  })
+  .forEach((testPath) => {
+    console.log(testPath);
```

Comment:
Could be moved into the if condition below so this only gets logged if the symlink doesn't exist yet. Otherwise it creates noise on every test run.

Reply:
Thanks, it's actually a leftover from development, so I'll remove it completely.

Reply:
Removed.